beater/beater.go

```diff
@@ -650,8 +649,14 @@ func (s *serverRunner) newFinalBatchProcessor(
 		if err != nil {
 			return nil, nil, err
 		}
-		monitoring.Default.Remove("libbeat")
-		monitoring.Default.Add("libbeat", s.libbeatMonitoringRegistry, monitoring.Full)
+		// We only want to restore the previous Libbeat registry if the output
+		// has a name, otherwise, keep the libbeat registry as is. This is to
+		// account for cases where the output config may be sent empty by the
+		// Elastic Agent.
```

Comment:
:100: for the comment. Stupid nit, could it either be `Libbeat` or `libbeat` in both places? I prefer `libbeat` cause lowercase is cool.